.github/workflows/check-merge.yml

```diff
@@ -62,11 +62,20 @@ jobs:
             --header 'content-type: application/json' \
             -d '["semver minor"]'
 
+
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.number }}
+          comment-author: 'github-actions[bot]'
+
       - name: Send PR review
         if: steps.find-blockers.outputs.found == 'true'
-        uses: peter-evans/create-or-update-comment@v2
+        uses: peter-evans/create-or-update-comment@v3
         continue-on-error: true
         with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
```

Comment:
what happens if there isn't a comment yet? Is this like undefined or something?

Reply:
I don't know the answer, but In the repo they have an example where they split create and update, I can use that one